EosGround/sendCommand.py

```diff
@@ -0,0 +1,41 @@
+
+from EosLib.packet.packet import Packet
+import EosLib.packet.definitions as PacketDefinitions
+
+import psycopg2
+import datetime
+
+conn = psycopg2.connect(
+    database="db_1", user='postgres', password='password', host='localhost', port='5432'
+)
+
+# Creating a cursor object using the cursor() method
+cursor = conn.cursor()
+
+
+def send_command():
+    time_sent = datetime.datetime.now()
```

Comment:
time_sent should not be set on insert

Reply:
see this comment.  time_sent should be sent when the radio driver passes off the packet to the Xbee hardware, not when you insert it into the DB here